depthai_demo.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python3.10
```

Comment:
I don't think this is a right assumption to make...

Reply:
agree, its a leftover from my testing, I decided to run demo via run_demo script which sources venv and calls depthai_demo.py with appropriate python